afdko/Tools/SharedData/FDKScripts/MakeOTF.py

```diff
@@ -1388,7 +1394,7 @@ def getROS(fontPath):
 	"""
 	R=O=S=None
 	fp = open(fontPath, "rb")
-	data = fp.read(5000)
+	data = str(fp.read(5000))
```

Comment:
again, wrong for the same reason I above. you're calling `str` on a bytes string, but it looks like you want to decode it.
Just open the file with `io.open` as text and pass the correct encoding. You'll read properly decoded (unicode) strings.

Reply:
see above